arbor/backends/multicore/threshold_watcher.hpp

```diff
@@ -52,7 +49,6 @@ class threshold_watcher {
     /// calling, because the values are used to determine the initial state
     void reset(const array& values) {
         values_ = values.data();
-        std::copy(values.begin(), values.end(), v_prev_.begin());
```

Comment:
is this safe? when testing we access the `v_prev_` array which would not be initialized.

Reply:
Errr... this should actually not be removed. Kind of scary that no test caught this.